server/index.js

```diff
@@ -13,27 +15,49 @@ app.listen(3000, () => {
 });
 
 app.get('/tetraflix/recommendations/:user', (req, res) => {
-  // const { user } = req.params;
-
-  // look up recs and cw for user by ID
-  // something like: queryResult = findOne({ where: { user: user} });
-
-  // very simplified result; actual will contain 20 recommendation
-  // and some number of currently watching movie objects (up to 20?);
-  const queryResult = '"{"recommendations":[{"id":8675309,"title":"Lord of the Rings","profile":{"action":30,"adventure":40,"fantasy":30},"progress":0},{"id":4879823,"title":"Finding Nemo","profile":{"animation":60,"family":40},"progress":0}],"currentlyWatching":[{"id":234523,"title":"Shrek","profile":{"animation":40,"comedy":30,"family":30},"progress":0.3},{"id":857636,"title":"Lego Batman","profile":{"action":30,"animation":40,"comedy":30},"progress":0.7}]}"';
-  res.send(queryResult);
+  const { user } = req.params;
+  mongoDb.UserMovies.findById(user, (err, doc) => {
```

Comment:
You're using a mix of promises and callbacks here. It might be cleaner if you refactored to use just promises.

Reply:
I also notice you did the same thing in the route below. If you refactor to use promises, then you only need one error handler, which is cleaner than having error handlers in multiple places.

Reply:
Done